packages/stripe_ios/ios/Classes/CardFieldView.swift

```diff
@@ -216,13 +216,39 @@ class CardFieldView: NSObject, FlutterPlatformView, STPPaymentCardTextFieldDeleg
             if let textErrorColor = cardStyle["textErrorColor"]  as? String {
                 cardField.textErrorColor = UIColor(hexString: textErrorColor)
             }
-            if let fontSize = cardStyle["fontSize"]  as? Int {
-                cardField.font = UIFont.systemFont(ofSize: CGFloat(fontSize))
-            }
+           
             if let placeholderColor = cardStyle["placeholderColor"]  as? String {
                 cardField.placeholderColor = UIColor(hexString: placeholderColor)
             }
             
+            let fontSize = CGFloat(cardStyle["fontSize"]  as? Int  ?? 18);
+            if let fontFamily = cardStyle["fontFamily"] as? String {
+                // Register font
+                registerFont(fontFamily)
+                let font =  UIFont(name: "\(fontFamily)", size: fontSize)
+                cardField.font = font ?? UIFont.systemFont(ofSize: fontSize)
+            } else {
+                cardField.font = UIFont.systemFont(ofSize: fontSize)
+            }
+        }
+    }
+    
+    func registerFont(_ fontFamily: String) {
+        let controller = UIApplication.shared.delegate?.window??.rootViewController as? FlutterViewController;
+        let bundle = Bundle.main
```

Comment:
what if the font is located in a separate package and not the main bundle? I do not think this solution facilitates for multi package projects. Either we should fix this or we should name this as a limitation

Reply:
I would prefer to push it like this, or without the registerFont. And then implement it also for packages and also on Android.

Reply:
let's ship it then :)